catroid/src/main/java/org/catrobat/catroid/content/bricks/GoToBrick.java

```diff
@@ -305,7 +306,9 @@ public boolean onTouch(View paramView, MotionEvent paramMotionEvent) {
 
 	@Override
 	public Brick clone() {
-		return new GoToBrick(destinationSprite);
+		GoToBrick copy = new GoToBrick(destinationSprite);
+		copy.spinnerSelection = spinnerSelection;
+		return copy;
```

Comment:
copied 
``` java
	private String touchPositionLabel;
	private String randomPositionLabel;
```
as well.

but IMHO this should be refactored when there is more time (WHY not use something static instead).